src/js/base/runtime.js

```diff
@@ -1230,7 +1224,6 @@ function (Namespace, jsnums, codePoint, util, exnStackParser, loader, seedrandom
     ************************/
     function checkType(val, test, typeName) {
       if(!test(val)) {
-        debugger;
```

Comment:
Hmmm, we like those debugger statements; they're sometimes the only way we can convince the browser to stop when we want it to...